### DISCUSSION TITLE
tensorboard description

### DISCUSSION THREAD

**Discussion by madey83** [idiap/coqui-ai-TTS#161] [Q&A] [answered]
is there any guide/document for  tensorboard?
 how to read all metrics, figures and graphs from it during model train?

**Comment by eginhard** [comment, marked as answer]
It depends on the model you're using. Best way to understand the losses is probably to read the research paper or the code. The original repository might also have some useful info in the issues and discussions, you can look there.

**Comment by madey83** [comment]
have you got some links for vits model?
